app/assets/javascripts/discourse/app/components/topic-map/topic-map-summary.gjs

```diff
@@ -10,6 +10,10 @@ import i18n from "discourse-common/helpers/i18n";
 import { avatarImg } from "discourse-common/lib/avatar-utils";
 
 export default class TopicMapSummary extends Component {
+  linksCount = this.args.topicDetails.links?.length ?? 0;
+  createdByUsername = this.args.topicDetails.created_by?.username;
+  lastPosterUsername = this.args.topicDetails.last_poster?.username;
```

Comment:
I wonder... could any of these change at runtime?

Even if not, I think they would make more sense as a getter. Using a field means we're copying the value into the component's state at initialization, which is not ideal. Less places storing state is always better!